mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -23,15 +23,23 @@ import org.apache.spark.Logging
 import org.apache.spark.mllib.linalg.Vectors
 import org.apache.spark.mllib.stat.{MultivariateStatisticalSummary, MultivariateOnlineSummarizer}
 import org.apache.spark.sql.DataFrame
-
 /**
  * Evaluator for regression.
  *
- * @param predictionAndObservations an RDD of (prediction, observation) pairs.
+ * @param predictionAndObservations an RDD of (prediction, observation) pairs,
+ * @param regThroughOrigin true if intercept is not included in linear regression model
```

Comment:
Both have pros and cons. I'm not sure which one is better from user's perspective (who is not reading the code). Personally, I would prefer `hasFitIntercept` to be consistent with the code. I can make this change if we can reach consensus.